www/js/control/ProfileSettings.jsx

```diff
@@ -215,42 +472,46 @@ const ProfileSettings = () => {
-               <SettingRow textKey="control.nuke-all" iconName="delete-forever" action={() => setNukeVis(true)}></SettingRow>
-               <SettingRow textKey={parseState(collectSettings.state)} iconName="pencil" action={() => setForceStateVis(true)}></SettingRow>
-               <SettingRow textKey="control.check-log" iconName="arrow-expand-right" action={showLog}></SettingRow>
-               <SettingRow textKey="control.check-sensed-data" iconName="arrow-expand-right" action={showSensed}></SettingRow>
-               <SettingRow textKey="control.collection" iconName="pencil" action={editCollectionConfig}></SettingRow>
-               <ControlDataTable controlData={collectSettings.config}></ControlDataTable>
-               <SettingRow textKey="control.sync" iconName="pencil" action={editSyncConfig}></SettingRow>
-               <ControlDataTable controlData={settings?.sync?.show_config}></ControlDataTable>
-               <SettingRow textKey="control.app-version" iconName="application" action={()=>console.log("")} desc={settings?.clientAppVer}></SettingRow>
-           </ExpansionSection>
-
-        {/* menu for "nuke data" */}
+            <ExpansionSection sectionTitle="control.dev-zone">
+                <SettingRow textKey="control.refresh" iconName="refresh" action={refreshScreen}></SettingRow>
+                <SettingRow textKey="control.end-trip-sync" iconName="sync-alert" action={endForceSync}></SettingRow>
+                <SettingRow textKey="control.check-consent" iconName="check" action={checkConsent}></SettingRow>
+                <SettingRow textKey="control.dummy-notification" iconName="bell" action={dummyNotification}></SettingRow>
+                {notifSchedule}
+                <SettingRow textKey="control.invalidate-cached-docs" iconName="delete" action={invalidateCache}></SettingRow>
+                <SettingRow textKey="control.nuke-all" iconName="delete-forever" action={() => setNukeVis(true)}></SettingRow>
+                <SettingRow textKey={parseState(collectSettings.state)} iconName="pencil" action={() => setForceStateVis(true)}></SettingRow>
+                <SettingRow textKey="control.check-log" iconName="arrow-expand-right" action={showLog}></SettingRow>
+                <SettingRow textKey="control.check-sensed-data" iconName="arrow-expand-right" action={showSensed}></SettingRow>
+                <SettingRow textKey="control.collection" iconName="pencil" action={editCollectionConfig}></SettingRow>
+                <ControlDataTable controlData={collectSettings.config}></ControlDataTable>
+                <SettingRow textKey="control.sync" iconName="pencil" action={editSyncConfig}></SettingRow>
+                <ControlDataTable controlData={syncSettings.show_config}></ControlDataTable>
+                <SettingRow textKey="control.app-version" iconName="application" action={()=>console.log("")} desc={appVersion}></SettingRow>
```

Comment:
nit + future: should we move this out of the developer zone? I am not sure why it needs to be hidden away.

It seems like fairly important information to know which version of the app you have, and it is surprisingly hard to figure our from other locations such as the play store. I think that the original thought by @sebastianbarry was that most control/settings screens hide away the app version under something like "General" (we should verify this by looking up the related commit)

However, after having deployed it for a while, I will point out that "Developer Zone" != "General". I would be OK with creating some other "versions" expansion section if needed, but I don't think it should be in the Developer Zone.